AmericanMedicalAssociation/ama-style-guide-2
# EWL-9201: Adjust alert banner styling for tablet.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- n/a

**Jira Ticket**
- [EWL-XXXX: Ticket Title](https://issues.ama-assn.org/browse/EWL-XXXX)

## Description
Adjusts styling for alert banner to accommodate tablet styling.


## To Test
- **NOTE** There is a pr in here to update the critical css for alert banner, check with this PR pulled or delete the critical css from the head tag when reviewing. (open console, find <style id="critical-css"></style> and delete)
- Link styleguide locally
- create new test banner
- confirm styling matches the following zeplins:

[Desktop](https://zpl.io/2pqDWmJ)

[Tablet](https://zpl.io/VD358vJ)

[Mobile](https://zpl.io/adoBnDe)

## Visual Regressions
-n/a

## Relevant Screenshots/GIFs
- n/a


## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.


## Additional Notes
**NOTE** There is a pr in here to update the critical css for alert banner, check with this PR pulled!!

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
